vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -185,6 +187,32 @@ - (BOOL)shouldAllowAdConversionTracking
   return true;
 }
 
+- (const std::string)getCountrySubdivision
+{
+  // Not needed on mobile
+  return "";
+}
+
+- (void)setCountrySubdivision:(const std::string &)countrySubDivision
+{
+  // Not needed on mobile
+}
+
+- (BOOL)didOverrideAdsSubdivision
+{
+  return true;
```

Comment:
Why ask client if this will always be true? These can just be returned in the `NativeAdsClient` instead of bridging to `BATBraveAds` (same with all the empty string & empty methods)

Reply:
I've made a few changes re this/your other comments for the iOS bits